lxd/storage/utils.go

```diff
@@ -1107,9 +1138,6 @@ func VolumeUsedByInstanceDevices(s *state.State, poolName string, projectName st
 			}
 
 			instStorageProject := project.StorageVolumeProjectFromRecord(&p, volumeType)
-			if err != nil {
```

Comment:
if this was a redundant error check can you remove it as a separate commit so we can backport this change independent of the feature?

Reply:
:+1: #14764